### PR TITLE
[feat] Add solution to "Stranger Code" (week1, web, 100 pts)

### PR DESCRIPTION
**Pull request by gteti**
Added week1 -> web -> Stranger Code challenge solution + fixing superlinter issues.

### CODE REVIEW

**Comment by gteti** [comment]
Telling for the second time just to be clear: This PR fixes also the Superlinter issue segnalated.
And I also started working on #7 here.

**Comment by gmacario** [comment]
> Telling for the second time just to be clear: This PR fixes also the Superlinter issue segnalated. And I also started working on #7 here.

@gteti please try to make atomic PRs. 

I have already fixed #7 in PR #10.

Could you cleanup this PR (or - better - create a new one) for the resolution of one challenge instead?

File: training/week1/miscellaneous/Dig_into_the_Blood/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ The image is in here: [Dig+Into+The+Blood.png](https://challenges.reply.com/tamt
 
 Use an _Image Editor_ (I used [Paint.NET](https://www.getpaint.net/)) to alter the image colors and then the **FLAG** will appear:
 
-```
+```verbatim
```
Comment (gmacario) [inline training/week1/miscellaneous/Dig_into_the_Blood/README.md]: Already handled in #10

File: training/week1/miscellaneous/Dig_into_the_Blood/flag.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+{FLG:TargetBuilding.dxf}
```
Comment (gmacario) [inline training/week1/miscellaneous/Dig_into_the_Blood/flag.txt]: Same

**Comment by gteti** [comment]
#10 was not there when I made this but ok about atomic PR. 
I've reverted the changes I made. This is an atomic PR now about the challenge.

**Comment by gteti** [comment]
There are no more conflicts.

File: training/week1/miscellaneous/dig-into-the-blood/README.md [hunk 3]
```diff
@@ -11,7 +11,11 @@ The image is in here: [Dig+Into+The+Blood.png](https://challenges.reply.com/tamt
 
 Use an _Image Editor_ (I used [Paint.NET](https://www.getpaint.net/)) to alter the image colors and then the **FLAG** will appear:
 
+<<<<<<< HEAD:week1/Dig_into_the_Blood/README.md
+```verbatim
+=======
 ```text
+>>>>>>> 00907800a5560f16e8a54984f036e419c9bde9b9:training/week1/miscellaneous/dig-into-the-blood/README.md
```
Comment (gmacario) [inline training/week1/miscellaneous/dig-into-the-blood/README.md]: Remove all those changes, the file is already OK in the main branch

**Review by gmacario — CHANGES_REQUESTED** [review]

File: training/week1/miscellaneous/dig-into-the-blood/README.md (hunk 3 above)
Comment (gteti) [reply training/week1/miscellaneous/dig-into-the-blood/README.md]: Done

**Review by gmacario — APPROVED** [review]

**Comment by gmacario** [comment]
LGTM, merged PR

**Pull request merged**